src/app/components/SwapWidget/style.scss

```diff
@@ -0,0 +1,107 @@
+.container {
+    background: #1C1C1C;
+    max-width: 600px;
+    box-shadow: 0 4px 6px rgba(0,0,0,0.1);
+    margin-top: 20px;
+}
+.header {
+    background: #212121;
+    padding: 24px 0 24px 0;
+    text-align: center;
+    width: 100%;
+    height: 100%;
+    min-height: 225px;
+    h1 {
+        font-size: 20px;
+    }
+    h3 {
+      font-size: 30px;
+      color: #fff;
+    }
+    h4 {
+      font-size: 20px;
+    }
+    button {
+        background: #212121;
+        cursor: pointer;
+        transition: all .2s ease-in-out;
+        padding: 0;
+        margin-top: 12px;
+        border: 1px solid rgba(61, 61, 61, 0.48);
+      &:hover {
+          background: rgba(255,255,255,.05)
+      }
+      &:active {
+        background: rgba(255,255,255,.05)
+      }
+    }
+    .asset {
+        display: inline-block;
+        vertical-align: middle;
+        width: 216px;
+        min-height: 102px;
+        line-height: 67px;
+        p {
+            position: relative;
+            bottom: 10px;
+            height: 0px;
+            margin: 0;
+        }
+    }
+    svg {
+        margin: 0px 16px;
+        vertical-align: middle;
+    }
+}
+.swap {
+    border: none !important;
+    margin: 0;
+    background: transparent !important;
+    &:hover {
+      opacity: .8;
+    }
+}
+.body {
+    padding-bottom: 30px;
+}
+.section {
+    margin:10px auto 0 auto;
+    max-width: 500px;
+    padding: 20px 16px;
+    min-height: 156px;
+    h5 {
+      font-size: 16px;
+    }
+    input {
```

Comment:
I would avoid adding custom styling to inputs because of design consistency. Feel free to tweak the global input styles in faast-ui if you like. Also, you can pass `inputClassName='flat'` to ReduxFormField to get rid of the drop-shadow